__tests__/app.test.js

```diff
@@ -277,10 +279,51 @@ describe("get articles by topic", () => {
   });
   test("404: should return not found", () => {
     return request(app)
-     .get("/api/articles?topic=banana")
-     .expect(404)
-     .then(({ body }) => {
+      .get("/api/articles?topic=banana")
+      .expect(404)
+      .then(({ body }) => {
         expect(body.msg).toBe(`ERROR: topic not yet created`);
       });
-  })
-})
+  });
+});
+
+describe("get comment count from article", () => {
+  test("200: should return the comment count", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        body.article.forEach((article) => {

```

Comment:
You shouldn't have to forEach here if you have more control over what you are sending back from the model. I think you are returning result.rows, I would return result.rows[0] because there should just be one article. Then you won't have to forEach here as you just have one article.